io.github.SkiffDesktopAdw.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "io.github.SkiffDesktopAdw",
```

Comment:
this app-id is invalid. github app-id contain 4 segments, the 3rd one being the associated project or username, the one the web pages are behind (if they exist)